src/redux.js

```diff
@@ -1161,7 +1167,11 @@ export function getPanelButtons(state) {
       next = { panel: "results", text: "Continue" };
     }
   } else if (state.currentPanel === "results") {
-    prev = { panel: "dataDisplayLabel", text: "Back" };
+    prev = isPanelEnabled(state, "dataDisplayLabel")
+      ? { panel: "dataDisplayLabel", text: "Back" }
+      : isPanelEnabled(state, "dataDisplayFeatures")
+      ? { panel: "dataDisplayFeatures", text: "Back" }
+      : null;
```

Comment:
Interesting use of nested ternary operators! I'm trying to understand why this was added. Does this go hand in hand with the changes in src/index.js?

Reply:
Good catch.  This was an opportunistic fix for the `hideSelectLabel` mode; I noticed when using the tool to build this new feature that it was still possible to get back to the label scene when it should not have been possible.